app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
+# This ApplicationHelper is a module that provides helper methods to be used across the application.
+# frozen_string_literal: true
+
 module ApplicationHelper
+  def full_title(page_title = "")
+    base_title = t("app_title")
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
+  end
```

Comment:
cú pháp của Ruby theo tiêu chí ngắn gọn, rõ ràng -> chỗ nào ko cần `()` thì em xóa đi nhé
```suggestion
  def full_title page_title = ""
    base_title = t "app_title"
    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
  end
```